resource_estimator/src/system/constants.rs

```diff
@@ -5,10 +5,10 @@
 pub const MAX_CODE_DISTANCE: u64 = 50;
 
 /// Maximum number of distillation rounds
-pub const MAX_DISTILLATION_ROUNDS: usize = 3;
+pub const MAX_DISTILLATION_ROUNDS: u64 = 3;
```

Comment:
Not sure if this is necessary. Usize seems to be more appropriate.

Reply:
This change increases consistency with other constants. Also, we are using `u64` mostly for job parameters.